### ISSUE TITLE
Admin Dashboard: "Disabling an App" is not implemented.

### ISSUE THREAD

**Issue by CamiMonte** [slashroots/harvest-v2#162]
The functionality to disable an app from the administrator's dashboard has not been implemented.

**Comment by paradoxxjm** [comment]
@CamiMonte was an UI proposal created for this feature?

**Comment by CamiMonte** [comment]
@paradoxxjm Yes, a UI proposal was created for this feature. The following describes the interactions as designed.

This feature can be accessed via:

1. Clicking the "Disable" link in the Apps tab (located in the "User Apps and App Activities" table) on the  users' Account Profile screen on the Admin's dashboard.
 _(See below High-Fidelity prototype demonstrating this interaction.)_

![user account profile - blank state](https://cloud.githubusercontent.com/assets/20387220/23175584/e6aa5fce-f82d-11e6-9310-9caa4955329d.png)
![user account profile - blank state](https://cloud.githubusercontent.com/assets/20387220/23175633/121819e4-f82e-11e6-93d5-735a3a80bb14.png)


2. Clicking a similar "Disable" link in the "Developer Apps" table.
_(See below Low-Fidelity Prototype demonstrating this interaction.)_

![disable app](https://cloud.githubusercontent.com/assets/20387220/23176227/35b10990-f830-11e6-9733-cc1ce1b549c0.png)
![disable app](https://cloud.githubusercontent.com/assets/20387220/23176240/44779228-f830-11e6-9b62-d590f8b5da72.png)
